src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -187,7 +188,7 @@ HttpStatusCode mongoUpdateContextSubscription
     /* The hasField checks are needed as lastNotification/count might not be present in the original doc */
     if (sub.hasField(CSUB_LASTNOTIFICATION))
     {
-      newSub.append(CSUB_LASTNOTIFICATION, getIntField(sub, CSUB_LASTNOTIFICATION));
+      newSub.append(CSUB_LASTNOTIFICATION, getLongField(sub, CSUB_LASTNOTIFICATION));
```

Comment:
Not sure but... maybe this is other case in which "CSUB_LASTNOTIFICATION might be either Int or Long ..." and we need the "dual read" you have implemented in other places to be safer?

(In fact, the "dual read" function seems is used in a lot of places, I'd suggest to move it to a separate and common function)